submissions/logistics_belts/logistics-belts.md

```diff
@@ -0,0 +1,27 @@
+
+## Description of a Logistics Belt
+
+A logistics belt is a somewhat complicated concept, but it's made much simpler when you see one in action. In this example, pay attention to how several items are sent along the same belt, each being loaded onto the belt only when requested. If iron and copper are all that are needed, no steel will be sent; likewise, if nothing is needed, the belt will remain empty.
+
+[embed logistics-belts-demo.mp4 here with the caption: A demonstration of logistics belts using circuit conditions, filter splitters, and loaders. Inserters can be used in lieu of loaders, but may need to be doubled or tripled depending on the belt speed.]
+
+You can see in the video that the circuit condition is set to admit materials when the destination has less than one hundred. In reality, the length of the belt will affect how many materials are sent; if the belt is long, one hundred will be buffered with hundreds still on the belt. There *is* a theoretical limit to how long the belt can be without overflowing the receiver chest, but in practice you won't hit it if you're using steel chests; at those distances, you'll be using trains instead.
+
+[embed logistics-belt-circuit-condition.jpg here with the caption: An example circuit condition for a logistics belt. Note that setting a limit of one hundred will actually send approximately 270 plates over this distance, due to plates remaining on the belt at time of cutoff.]
+
+## Use Cases for Logistics belts
+
+Logistics belts are useful when you need a lot of an item all at once, and then none of it for a very long time. Anyone who has ever automated belts with bots will be familiar with the phenomenon known as the "charging cloud" that comes about when your logistics network asks for no less than ten thousand gears and fifteen hundred electronic circuits. Likewise, when you ask for something like the materials to make a nuclear reactor or a rocket silo, bots will tend to swarm around an assembler in an area that was never designed for that degree of throughput. When you need a lot of an item, it's much easier to get it in by belt: sending ten thousand gears takes less than four minutes and barely damages your UPS; with bots it would take 2500 separate trips and likely make that many bots go airborne, swamping charging capacity that may never have been designed for that and taking forever to return to equilibrium. Running dedicated belts for every item, however, creates its own set of problems; often, the belts will be sitting idle and taking up space within your factory, and you may not even have room to run an eight-bus in the first place. For something like your belt mall, sending a large load of gears intermixed with a large load of electronic circuits is trivial for a logistics belt; once they arrive in that location, using bots internally is much less strain on your logistics network.
```

Comment:
Could you add a larger scale example of the logistic belt concept in action here? You've done a great job explaining the basic but seeing where it can go would be super interesting.